_summaries/chen2020a_1.md

```diff
@@ -1,19 +1,40 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {chen2020a_1}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
+* What is the core idea? <br/>
+This paper comes up with a simple framework for contrasive learning of visual representation, which not only archieves state of the art accuracy, but also has simplier structure. The result of SimCLR is shown in the following picture <br/>
+<img src = "chen2020_a_3.png">
+
+* How is it realized (technically)? <br/>
+The architecture of the SimCLR is shown as the following picture <br/>
+<img src = "chen2020_a_1.png">
+As you can see from the picture. First, for each data sample, x, the model generates two views of of the data sample by sequentially applying three data augmentation methods -- random cropping, random color distortions, and random Gauessian blur. Then the model applies encoder $$f(-)$$. There are various choices of the encoder, but in this paper, the authors use ResNet. Projection head $$g(-)$$ is MLP with one hidden layer, and it uses ReLU as activation function. The reason we need projection head $$g(-)$$ is that it maps representations to the space where contrasive loss can be applied. <br/>
+
+The training process is shown as following. <br/>
+<img src = "chen2020_a_2.png">
+
```

Comment:
Elaborate a bit on the contrastive loss. What inputs should be "similar" in the latent space, and which ones shouldn't?

Reply:
Could be worth mentioning the optimizer they use since it's not the typical ADAM/SGD. Also, how long does their network take to train?

Reply:
I see, thanks, that's very inspiring.